combat-trainer.lic

```diff
@@ -222,9 +224,6 @@ class LootProcess
     @necro_heal = thanatology['heal'] || false
     echo("  @necro_heal: #{@necro_heal}") if $debug_mode_ct
 
-    @necro_train_first_aid = thanatology['train_first_aid'] || false
```

Comment:
https://github.com/rpherbig/dr-scripts/blob/master/validate.lic#L414-L419 is probably pretty close to what you'd want